crates/aiken-lang/src/tipo/expr.rs

```diff
@@ -888,6 +891,33 @@ impl<'a, 'b> ExprTyper<'a, 'b> {
                     unmatched,
                 });
             }
+        } else if !value_is_data
+            && self
+                .environment
+                .check_exhaustiveness(
+                    vec![pattern.clone()],
+                    collapse_links(value_typ.clone()),
+                    location,
+                )
+                .is_ok()
+        {
+            self.environment
+                .warnings
+                .push(Warning::SingleConstructorExpect {
+                    location: Span {
+                        start: location.start,
+                        end: location.start + 6,
```

Comment:
What is `6` here? `|expect|`? Could we capture this as a variable / constant ?

Reply:
we cannot. we don't capture the location of the keyword in the AST, just the whole assignment, the pattern and the right hand side of the `=`